python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -403,6 +403,35 @@ class IsrTaskLSSTConfig(pipeBase.PipelineTaskConfig,
         target=MaskingTask,
         doc="Masking task."
     )
+    doE2VEdgeBleedMask = pexConfig.Field(
+        dtype=bool,
+        doc="Mask flag-like edge bleeds from saturated columns "
+            "in E2V amplifiers.",
+        default=True,
+    )
+    e2vEdgeBleedSatMinArea = pexConfig.Field(
+        dtype=int,
+        doc="Minimum limit of saturated cores footprint area to apply edge"
+            "bleed masking in E2V amplifiers.",
+        default=20000,
+    )
+    e2vEdgeBleedSatMaxArea = pexConfig.Field(
+        dtype=int,
+        doc="Maximum limit of saturated cores footprint area to apply edge"
+            "bleed masking in E2V amplifiers.",
+        default=100000,
+    )
+    e2vEdgeBleedYMax = pexConfig.Field(
+        dtype=int,
+        doc="Height of edge bleed masking in E2V amplifiers (width is the"
+            "width of the amplifier).",
+        default=350,
+    )
+    e2vEdgeBleedThreshold = pexConfig.Field(
+        dtype=float,
+        doc="Sky background threshold for E2V edge bleed detection.",
```

Comment:
Do you mean the sky background threshold, or the threshold above sky?  And what are the units (e- or adu)?